pyproject.toml

```diff
@@ -1,62 +1,65 @@
-  { name = "Guilherme Crocetti" },
-  { name = "Alexander Shadchin" },
-  { name = "Lieser Fernández Gallo", email = "leiserfg@gmail.com" },
+  { name = 'Kareem Moussa' },
+  { name = 'Brandon Milton' },
+  { name = 'Fabio Rosado' },
+  { name = 'Niko Eckerskorn' },
+  { name = 'Or Carmi' },
+  { name = 'George Kontridze' },
+  { name = 'Sean Chambers' },
+  { name = 'Nils Philippsen' },
+  { name = 'Alexander Duryagin' },
+  { name = 'Sakorn Waungwiwatsin' },
+  { name = 'Jacob Floyd' },
+  { name = 'Guilherme Crocetti' },
+  { name = 'Alexander Shadchin' },
+  { name = 'Lieser Fernández Gallo', email = 'leiserfg@gmail.com' },
 ]
-license = { file = "LICENSE" }
-keywords = ["http", "api", "rest", "client", "retrofit"]
-urls = { Repository = "https://github.com/prkumar/uplink" }
+license = { file = 'LICENSE' }
+keywords = ['http', 'api', 'rest', 'client', 'retrofit']
+urls = { Repository = 'https://github.com/prkumar/uplink' }
 classifiers = [
-  "Development Status :: 4 - Beta",
-  "Intended Audience :: Developers",
-  "License :: OSI Approved :: MIT License",
-  "Operating System :: OS Independent",
-  "Programming Language :: Python :: 3.10",
-  "Programming Language :: Python :: 3.11",
-  "Programming Language :: Python :: 3.12",
-  "Programming Language :: Python :: 3.13",
-  "Programming Language :: Python :: 3 :: Only",
-  "Programming Language :: Python :: Implementation :: CPython",
-  "Programming Language :: Python :: Implementation :: PyPy",
-  "Topic :: Software Development :: Libraries",
+  'Development Status :: 4 - Beta',
+  'Intended Audience :: Developers',
+  'License :: OSI Approved :: MIT License',
+  'Operating System :: OS Independent',
+  'Programming Language :: Python :: 3.10',
+  'Programming Language :: Python :: 3.11',
+  'Programming Language :: Python :: 3.12',
+  'Programming Language :: Python :: 3.13',
+  'Programming Language :: Python :: 3 :: Only',
+  'Programming Language :: Python :: Implementation :: CPython',
+  'Programming Language :: Python :: Implementation :: PyPy',
+  'Topic :: Software Development :: Libraries',
 ]
 
-requires-python = ">=3.10"
-dependencies = ["requests>=2.18.0", "six>=1.13.0", "uritemplate>=3.0.0"]
+requires-python = '>=3.10'
+dependencies = ['requests>=2.18.0', 'six>=1.13.0', 'uritemplate>=3.0.0']
+dynamic = ['version']
 
 [dependency-groups]
 test = [
-  "pytest",
-  "pytest-mock",
-  "pytest-cov",
-  "pytest-twisted",
-  "pytest-asyncio",
+  'pytest',
+  'pytest-mock',
+  'pytest-cov',
+  'pytest-twisted',
+  'pytest-asyncio',
 ]
-marshmallow = ["marshmallow>=2.15.0"]
-pydantic = ["pydantic>=2.0.0"]
-aiohttp = ["aiohttp>=3.8.1"]
-twisted = ["twisted>=21.7.0"]
+marshmallow = ['marshmallow>=2.15.0']
+pydantic = ['pydantic>=2.0.0']
+aiohttp = ['aiohttp>=3.8.1']
+twisted = ['twisted>=21.7.0']
 
 [build-system]
-requires = ["hatchling"]
-build-backend = "hatchling.build"
+requires = ['hatchling']
+build-backend = 'hatchling.build'
 
-[tools.pytest]
+[tool.hatch.version]
```

Comment:
Do you think we need this? isn't better to have the version here in the pyproject? If the version is needed at runtime it can be gathered with 
```python
from importlib.metadata import version
version("poetry")
```

Reply:
Good point! I wasn't aware of `importlib.metadata` package. Keeping version in pyproject again, and updated `__version__` to use `importlib.metadata.version`.